test/externalTests/ens.sh

```diff
@@ -35,7 +35,7 @@ function ens_test
     export OPTIMIZER_LEVEL=1
     export CONFIG="truffle-config.js"
 
-    truffle_setup https://github.com/solidity-external-tests/ens.git upgrade-0.8.0
+    truffle_setup https://github.com/solidity-external-tests/ens.git master
```

Comment:
Do we really want to change this? For the other scripts, it's usually off branch like upgrade-0.8.0 or develop-0.8.0

Reply:
Yeah, we usually run those tests on two different versions (separate for `develop` and for `breaking`) so having separate branches for them makes sense in my opinion, even if one or both sometimes just point at `master`.

BTW, why was it 0.8.0 in `develop`? I think it should still be the 0.7.0 branch.

Reply:
Makes sense to me. Although it is a bit confusing to have branches and have the branches merged to master as well.

Reply:
I missed one thing there. This is a PR to `develop` not ` breaking`. So I think we should be running on the commit `master` is now on after all. But we should just create our own `master_070` branch pointing at the same commit so that we can add changes there if any are needed for compatibility without having to update scripts in solidity repo.

Similarly, the `upgrade-0.8.0` branch should be renamed to `master_080` to match the convention we use in other repos. I.e. the name of the original branch it's based on plus the compiler version it's meant for.

Reply:
After looking into the repo I see that `upgrade-0.8.0` actually points at the same commit as `master`. I thought it had some extra changes for 0.8.0. I thought that rebasing this PR on `breaking` would fix it but apparently not.

Anyway, I left these branches intact but created `master_070` and `master_080` next to them to follow the convention from other repos. My suggestion would be to drop the `upgrade-0.8.0` branch and also to put `master` back to where it is in the upstream repo so that we can pull into it regularly and then rebase our branches on it.

I also removed all branches other than `master` that were not created by us. We don't have them in other repos either and I think this makes it easier to see how our repo differs from the relevant branches in the upstream repo.